index.d.ts

```diff
@@ -126,6 +126,8 @@ interface ClientOptions {
     password?: string;
   };
   disablePrototypePoisoningProtection?: boolean | 'proto' | 'constructor';
+  responseMaxHeapPercentage?: number;
```

Comment:
nit: i still think we can make this an object since `responseMaxHeapPercentage` is not used unless `memoryCircuitBreakerEnabled`, is there a specific reason why it can't?

Reply:
will do